### PR TITLE
Docs: Move Migrations out of Extensions

### PR DESCRIPTION
**Pull request by phazonoverload**
Docs for #20626

### CODE REVIEW

**Review by paescuj — COMMENTED** [review]
Awesome, thank you! ❤️ 

Just a small change request:
This newly introduced config option `MIGRATIONS_PATH` has a default of `./migrations`, so there's usually no need to explicitly define the option, migrations can just be placed in this folder.
With the defaults configs, this means the folder is expected to live at the same level as the `extensions` folder:
```
├── extensions
└── migrations
```

File: .changeset/beige-toys-serve.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"docs": patch
+---
+
+Added a note about the relocation of custom migrations
```
Comment (paescuj) [inline .changeset/beige-toys-serve.md:5]:
```suggestion
Added a note about / reflect the relocation of custom migrations
```

**Comment by phazonoverload** [comment]
How's that @paescuj

File: docs/releases/breaking-changes.md [hunk 2]
```diff
@@ -159,6 +159,14 @@ If your current workflow depends on redirecting to an external domain after succ
 
 `AUTH_<PROVIDER>_REDIRECT_ALLOW_LIST` accepts a comma-separated list of URLs (path is included in comparison).
 
+### Moved Migrations Out of Extensions
+
+Migrations are no longer considered an extension type as of this release. The `migrations` extensions directory must be
+removed.
```
Comment (paescuj) [inline docs/releases/breaking-changes.md:165]:
```suggestion
migrated.
```

**Review by paescuj — APPROVED** [review]
> How's that @paescuj

Perfect 😘👌

**Pull request merged**